draft-ietf-tls-rfc8446bis.md

```diff
@@ -4102,6 +4103,13 @@ decrypt_error:
   to correctly verify a signature or validate a Finished message
   or a PSK binder.
 
+ticket_invalid:
+: The server received a ticket that is invalid and is necessary
+to complete the handshake. For instance, the server might
+be configured to require PSK mode. Note that if a ticket
+is being used for session resumption, the server MAY
+ignore the ticket and perform a full handshake instead.
```

Comment:
Do you really want 2119 language here?  This is just how session resumption is supposed to work.

```suggestion
is being used for session resumption, the server is expected
to ignore the ticket and perform a full handshake instead.
```